spinalcordtoolbox/scripts/sct_warp_template.py

```diff
@@ -245,18 +235,31 @@ def get_parser():
     return parser
 
 
+S_DEPRECATION_STRING = """\
+As of SCT v6.1, probabilistic spinal levels have been replaced with a single integer spinal level file, \
+which can be found inside of the warped 'template/' folder. The '-s' option is no longer \
+needed.
+
+For more information on the rationale behind this decision, please refer to:
+  - https://github.com/spinalcordtoolbox/PAM50/issues/16
+  - https://forum.spinalcordmri.org/t/updating-spinal-levels-feedback-needed/1136\
+"""
+
+
 def main(argv: Sequence[str]):
     parser = get_parser()
     arguments = parser.parse_args(argv)
     verbose = arguments.v
     set_loglevel(verbose=verbose)
 
+    if arguments.s:
+        raise parser.error(S_DEPRECATION_STRING)
```

Comment:
Nitpick: [`parser.error`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error) doesn't return an exception that can be raised; it exits the program.
```suggestion
        parser.error(S_DEPRECATION_STRING)
```